rfcs/20211118-support-implicit-concat/README.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Support of implicit concat
+
+## Introduction
+
+This document explores a way of supporting implicit concat. Implicit concat is a
+concat primitive which is done without any additional copies just by storing
+correspondent parts into right spots of final tensor. Such approach allows to
+save at least one memory pass over the final output. Sometimes it may be two or
+more.
```

Comment:
Not sure about typical granularity of RFC, but I think it is necessary to state the list of primitives that will support implicit concat.

Reply:
I will add a section with coverage. Since we are quite resource-constrained and the feature is complex enough just to add everywhere, we will extend support as needed upon request, but will start with convolution and pooling coverage with strided input and output. Thanks.